CHANGELOG.md

```diff
@@ -122,6 +123,10 @@ entirely to allow inference to fill in the type.
       quickly than you can with dart2js. In order to do that, run
       `pub serve test --web-compiler=dartdevc`, and then run
       `pub run test -p chrome --pub-serve=8080`.
+      
+      Note that `dart2js` is still the compiler you will use for production:
```

Comment:
This is a bit too strongly worded I think, maybe "Note that dart2js is still the recommended compiler for production"